index.js

```diff
@@ -3104,7 +3104,7 @@
   def('trim',
       {},
       [$.String, $.String],
-      R.trim);
+      function(s) { return s.trim(); });
```

Comment:
Actually that's a very good point. I hadn't thought of it like that. Maybe we should hold off then.